src/js/twitter.js

```diff
@@ -8,5 +67,6 @@
   js = d.createElement(s)
   js.id = id
   js.src = p + '://platform.twitter.com/widgets.js'
+  js.setAttribute('onload', twitter_filter(id))
   fjs.parentNode.insertBefore(js, fjs)
```

Comment:
Calling with `id` is not relevant now?

Reply:
Removed id from the function call - wish I'd checked to see if that was actually the id of the widget - caused me a lot of trouble